### PR TITLE
feat: bump major version

### PR DESCRIPTION
**Pull request by schabibi1**
BREAKING CHANGE: remove axios

### CODE REVIEW

**Comment by schabibi1** [comment]
We're waiting for [storyblok-js-client](https://github.com/storyblok/storyblok-js-client), [storyblok-js](https://github.com/storyblok/storyblok-js), and [storyblok-react](https://github.com/storyblok/storyblok-react) to solve timeout.

After that quick fix, we'll update the [storyblok-react](https://github.com/storyblok/storyblok-react) version in [gatsby-source-storyblok](https://github.com/storyblok/gatsby-source-storyblok).

File: lib/vite.config.ts [hunk 1]
```diff
@@ -13,11 +13,14 @@ export default defineConfig(() => {
...
-        external: ["react", "axios"],
+        external: ["react"],
         output: {
           globals: { react: "React" },
         },
       },
     },
+    test: {
+      setupFiles: ["./setup.js"],
+    },
```
Comment (alexjoverm) [inline lib/vite.config.ts:24]: We don't need this. This was for Vitest but we don't use it here

File: package.json [hunk 2]
```diff
@@ -61,5 +61,8 @@
     "tabWidth": 2,
     "semi": true,
     "singleQuote": false
+  },
+  "dependencies": {
+    "isomorphic-fetch": "^3.0.0"
   }
```
Comment (alexjoverm) [inline package.json:67]: Remove this, this should be only on lib/package.json

**Review by alexjoverm — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (fgiuliani) [reply package.json:67]: What would be the `isomorphic-fetch` used for?

File: lib/vite.config.ts (hunk 1 above)
Comment (fgiuliani) [reply lib/vite.config.ts:24]: I asked @schabibi1 about this line on private 😄

Comment (schabibi1) [reply lib/vite.config.ts]: Done @alexjoverm ✅ @fgiuliani Responded 😉

File: package.json (hunk 2 above)
Comment (schabibi1) [reply package.json]: @alexjoverm Removed✅ @fgiuliani We implemented support for the use case without Fetch API support as we imagined such requests would come from users.

**Review by fgiuliani — APPROVED** [review]

**Pull request merged**